cli/ballerina-cli/src/main/resources/cli-help/ballerina-build.help

```diff
@@ -1,75 +1,60 @@
 NAME
-       ballerina-build - Build the Ballerina package or standalone file and generate an executable
+       ballerina-build - Compiles the current package
```

Comment:
Is it correct to rephrase it like this? Considering it's applicable to single files as well and we do speak of those here?

Reply:
I think it's okay. I did it intentionally. :) 

I explained the single file case in the description.